lib/PuppeteerSharp/Page.cs

```diff
@@ -43,6 +43,9 @@ public class Page : IDisposable
         private readonly Dictionary<string, Worker> _workers;
         private readonly ILogger _logger;
         private bool _ensureNewDocumentNavigation;
+        private PageGetLayoutMetricsResponse _metrics;
```

Comment:
I think this should be renamed to `_burstModeMetrics` or similar
```suggestion
        private PageGetLayoutMetricsResponse _burstModeMetrics;
```